starter-code/css/main.css

```diff
@@ -1,8 +1,168 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    margin:0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Varela Round', sans-serif;
+    /*line-height: 3.5em;*/
+    color:#1A1A1A;
+}
+
+.nav-bar {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color:#FFF;
+    width: 100%;
+
+}
+
+.nav-bar ul{
+    width: 40%;
+    margin-right: 20px;
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-bar ul li{
+    padding: 2px;
+    font-size: 1em;
+}
+
+.nav-bar div img {
+    width: 125px;
+}
+
+.conteiner {
+    background-image:url("./../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: transparent;
+    height: 650px;
```

Comment:
If you want the ".conteiner" to take the whole screen, as in a landing page, you could set the height to be 100vh, the "vh" unit being the size of the viewport, whether it's on mobile or desktop, it would be 100vh of the device's viewport.

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

You could learn more about the css relative units in the relative section of this page.